frontend/public/components/routes.tsx

```diff
@@ -133,8 +207,17 @@ const TLSSettings = props => <span>
 const RouteDetails: React.SFC<RoutesDetailsProps> = ({obj: route}) => <React.Fragment>
   <div className="co-m-pane__body">
     <ResourceSummary resource={route} showPodSelector={false} showNodeSelector={false}>
+      <dt>Hostname</dt>
+      <dd>{getRouteHost(route, false)}</dd>
```

Comment:
I think this should just be `route.spec.host`

Reply:
R u sure?  `getRouteHost()` finds the oldest Admitted Ingress.host, as opposed to just using `route.spec.host`.

Reply:
Yeah, that's what I want to avoid... I want to show the current requested hostname. Location shows the admitted value.

Reply:
ok, fixed